DTAdmin/Admins/NetworkManager.swift

```diff
@@ -0,0 +1,255 @@
+//
+//  NetworkManager.swift
+//  DTAdmin
+//
+//  Created by ITA student on 10/15/17.
+//  Copyright © 2017 Yurii Krupa. All rights reserved.
+//
+
+import Foundation
+
+class NetworkManager {
+  let urlProtocolPrefix = "http://"
+  let urlToHost = "vps9615.hyperhost.name"
+  let urlSuffixToUserLogIn = "/login/index"
+  let urlSuffixToUserLogOut = "/login/logout"
+  let urlSuffixToAdmins = "/AdminUser"
+  let urlSuffixToGetRecords = "/getRecords"
+  let urlSuffixToInsertData = "/insertData"
+  let urlSuffixToGetRecordById = "/getRecords"
+  let urlSuffixToUpdateRecord = "/update"
+  let urlSuffixToDeleteRecord = "/del"
```

Comment:
I like this approach with constants. You may consider using enum with associated type to make it better